Vuologist/bumpie
# Feat: add aws sign up call to sign in page

closes #76 

**Test Instructions:**

**Test 1**
- open chrome console
- enter a random email to the email field
- enter a password such as Password1@ to password field
- enter a same password to confirm password field

if everything worked then you should see nothing.

**Test 2**
- open chrome console
- enter a random email to the email field
- enter a password such as password1@ to password field
- enter a same password to confirm password field

if everything worked then you should see an alert that password does not meet password requirement.
